package.json

```diff
@@ -31,6 +31,7 @@
     "@babel/preset-env": "7.0.0-beta.53",
     "@babel/preset-react": "7.0.0-beta.53",
     "@babel/runtime": "7.0.0-beta.53",
+    "@wordpress/block-serialization-spec-parser": "1.0.0",
```

Comment:
Oh I suppose shrinkwrap didn't update since this is already a sub-dep of `@wordpress/blocks`?

Reply:
That's right, I updated this manually during the rebase.